.travis.yml

```diff
@@ -108,13 +96,55 @@ jobs:
        - chmod +x ./trillian/integration/integration_test.sh
        - ./trillian/integration/integration_test.sh
        - popd
+    - name: "presubmit"
+      script:
+       - ./scripts/presubmit.sh --no-linters --no-generate
+      env:
```

Comment:
This would be slightly easier to read if env came before script

Reply:
Done.